libs/astx/tests/test_comprehensions.py

```diff
@@ -30,8 +30,54 @@ def test_list_comprehension() -> None:
             )
         ],
     )
+    print(list_compre)
+    repr(list_compre)
+    assert str(list_compre)
+    assert list_compre.get_struct()
+    assert list_compre.get_struct(simplified=True)
+    visualize(list_compre.get_struct())
+
+
+def test_generator_expr() -> None:
+    """Test `GeneratorExpr` class with conditions of Iterable type."""
+    comp_1 = astx.ComprehensionClause(
+        target=astx.Variable("x"),
+        iterable=astx.Variable("List"),
```

Comment:
not totally wrong, but not too much pythonic, we would usually use something like "mylist" or "my_list", a variable usually follow the snake case convention.